src/libraries/Microsoft.PowerFx.Connectors/Public/CdpDelegationInfo.cs

```diff
@@ -10,7 +10,7 @@ internal class CdpDelegationInfo : TableDelegationInfo
     {
         public override ColumnCapabilitiesDefinition GetColumnCapability(string fieldName)
         {
-            if (ColumnsCapabilities.TryGetValue(fieldName, out ColumnCapabilitiesBase columnCapabilitiesBase))
+            if (ColumnsCapabilities.TryGetValue(fieldName, out ColumnCapabilitiesBase columnCapabilitiesBase) && columnCapabilitiesBase != null)
```

Comment:
Is adding a `null` value in the ColumnsCapabilities dictionary a valid scenario? It doesn't feel like it should be. If that is not the case, then we should block this in the addition, not in the retrieval. If that's a valid scenario, then this fix looks good.

Reply:
I think having missing capabilities is allowed and we shouldn't throw
This is clearly suboptimal when you don't have capabilities (as this will prevent us from delegating) but I don't think we're hiding a bug

Reply:
Let me remove that part